### PR TITLE
docs(README): improve documentation.

### PR DESCRIPTION
**Pull request by Ryssa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,28 +49,40 @@ When installing this package for the first time, the following shareable configu
...
+- **GH_TOKEN**: A GitHub [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token);
+
 - **RELEASE_DOCS_PROJECT_ID**: Unique project identifier. It will be used as the changelog file title on the S3 bucket;
 
 - **RELEASE_DOCS_AWS_ACCESS_KEY_ID**: [AWS access key](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) from the S3 bucket where the changelog file will be hosted; 
 
 - **RELEASE_DOCS_AWS_SECRET_ACCESS_KEY_ID**: [AWS secret access key](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) from the S3 bucket where the changelog file will be hosted.
 
-### Trigger a release locally
+### Triggering a Release Locally
```
Comment (Artfloriani) [inline README.md:74]:
```suggestion
### Triggering a Releasing Locally
```

File: README.md [hunk 2]
```diff
@@ -47,28 +49,40 @@ When installing this package for the first time, the following shareable configu
...
 
 ```bash
-$ RELEASE_DOCS_PROJECT_ID=<project ID> RELEASE_DOCS_AWS_ACCESS_KEY_ID=<AWS access key> RELEASE_DOCS_AWS_SECRET_ACCESS_KEY_ID=<AWS secret access key> RELEASE_DOCS_AWS_BUCKET=<AWS bucket> GH_TOKEN=<GitHub token> semantic-release --dry-run=false --no-ci
+$ RELEASE_DOCS_PROJECT_ID=<project ID> RELEASE_DOCS_AWS_ACCESS_KEY_ID=<AWS access key> RELEASE_DOCS_AWS_SECRET_ACCESS_KEY_ID=<AWS secret access key> RELEASE_DOCS_AWS_BUCKET=<AWS bucket> GH_TOKEN=<GitHub token> npm run semantic-release --dry-run=false --no-ci
 ```
 
-### Automation with CI
+### Automation With CI
```
Comment (Artfloriani) [inline README.md:80]:
```suggestion
### Running in CI
```

File: README.md [hunk 3]
```diff
@@ -47,28 +49,40 @@ When installing this package for the first time, the following shareable configu
...
 
-First, setup all the required [environment variables](#Configuration) on your CI environment. Then just add the following script to your CI configuration file:
+First, setup all the required [environment variables](#Configuration) on your CI environment. Then just call the following script on your CI configuration file to trigger the release process:
```
Comment (Artfloriani) [inline README.md:82]:
```suggestion
First, setup all the required [environment variables](#Configuration) on your CI environment. Then just call the following script in your CI configuration file to trigger the release process:
```

File: README.md (hunk 1 above)
Comment (Artfloriani) [reply README.md]: I think I mispelled here @Ryssa: 

> Triggering a Releasing Locally

I think I wanted to say

> Releasing Locally

Sorry :(

File: README.md [hunk 4]
```diff
@@ -47,28 +49,40 @@ When installing this package for the first time, the following shareable configu
...
+- **GH_TOKEN**: A GitHub [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token);
+
 - **RELEASE_DOCS_PROJECT_ID**: Unique project identifier. It will be used as the changelog file title on the S3 bucket;
 
 - **RELEASE_DOCS_AWS_ACCESS_KEY_ID**: [AWS access key](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) from the S3 bucket where the changelog file will be hosted; 
 
 - **RELEASE_DOCS_AWS_SECRET_ACCESS_KEY_ID**: [AWS secret access key](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) from the S3 bucket where the changelog file will be hosted.
 
-### Trigger a release locally
+### Triggering a Releasing Locally
```
Comment (Ryssa) [inline README.md:74]:
```suggestion
### Releasing Locally
```

**Pull request merged**